src/Features/Core/CodeFixes/CodeFixService.cs

```diff
@@ -261,8 +262,10 @@ internal partial class CodeFixService : ICodeFixService
                 // this can happen for suppression case where all diagnostics can't be suppressed
                 return result;
             }
+
             var extensionManager = document.Project.Solution.Workspace.Services.GetService<IExtensionManager>();
-            var fixes = await extensionManager.PerformFunctionAsync(fixer, () => getFixes(diagnostics)).ConfigureAwait(false);
+            var fixes = await extensionManager.PerformFunctionAsync(fixer, () => getFixes(diagnostics), defaultValue: SpecializedCollections.EmptyEnumerable<CodeFix>()).ConfigureAwait(false);
```

Comment:
That's true @heejaechang but I decided to explicitly return empty collection to avoid any possible problems due to missing null check :)